Android/app/src/main/java/objects/Weather.java

```diff
@@ -0,0 +1,39 @@
+package objects;
+
+public class Weather {
+
+    private int mImageId;
+    private int mMinTemp;
+    private int mMaxTemp;
+    private String mDayOfWeek;
+    private String mDate;
+
+    public Weather(int imageId, int maxTemp, int minTemp, String dayOfWeek, String date) {
+        this.mImageId = imageId;
+        this.mMaxTemp = maxTemp;
+        this.mMinTemp = minTemp;
+        this.mDayOfWeek = dayOfWeek;
+        this.mDate = date;
+    }
+
+    public int getmMinTemp() {
```

Comment:
change the name of getters & setters to remove `m` before the variable names, like
this should be `getMinTemp`